aspnetcore/security/docker-https.md

```diff
@@ -27,7 +27,10 @@ The [.NET Core 2.2 SDK](https://dotnet.microsoft.com/download) or later is requi
 
 ## Certificates
 
-A certificate from a [certificate authority](https://wikipedia.org/wiki/Certificate_authority) is required for [production hosting](https://blogs.msdn.microsoft.com/webdev/2017/11/29/configuring-https-in-asp-net-core-across-different-platforms/) for a domain. [Let's Encrypt](https://letsencrypt.org/) is a certificate authority that offers free certificates.
+A certificate from a [certificate authority](https://wikipedia.org/wiki/Certificate_authority) is required for [production hosting](https://blogs.msdn.microsoft.com/webdev/2017/11/29/configuring-https-in-asp-net-core-across-different-platforms/) for a domain. The following certificate authorities offer free certificates:
+
+* [Let's Encrypt](https://letsencrypt.org/)
+* [OpenSSL](https://www.openssl.org/)
```

Comment:
OpenSSL is not a certificate authority. It's a toolkit you can use to create one, and it doesn't create ones that are generally suitable for production, due to their self-signed nature. So this line does not belong.

Reply:
Perhaps we can mention this comment from the perspective of using OpenSSL generated certs for local testing then?

Reply:
@blowdart - Went ahead and gave this a pass on clarifying for OpenSSL.

cc: @Rick-Anderson

Reply:
We don't mention any other tools for test generation, other than our own one, for which you don't need OpenSSL.

I don't see why mentioning it at all is needed.

Reply:
If you do need to mention it, then really there should be an entire page on generating self signed certificates, using powershell, the .net developer toolset and openssl. Which isn't a bad idea :)

Reply:
Incidentally, I did come across using OpenSSL certs with istio.  One of the bits involved using aspnetcore containers, which prompted these thoughts here to at least make sure that the .pfx file was used instead of .crt.

As for a separate doc, that sounds interesting as well.  I'm assuming you may have some initial scripts too? :)

Reply:
> there should be an entire page on generating self signed certificates, using powershell, the .net developer toolset and openssl. Which isn't a bad idea :)

That would go in the .NET repo.  @BillWagner  can you recommend where that should go?

Reply:
@Rick-Anderson / @blowdart, perhaps we can use a separate pr in reference to the following?

> there should be an entire page on generating self signed certificates, using powershell, the .net developer toolset and openssl. Which isn't a bad idea :)

@BillWagner - can you let us know if you're able to find a good landing spot for using self-signed certs as @blowdart mentions?

Reply:
Since we are discussing all three options (OpenSSL, powershell, `dotnet dev-certs`), we should put that article under [this overview](https://docs.microsoft.com/dotnet/core/additional-tools/).  Note that we already have an article on cert creation in the [ASP.NET Core docs](https://docs.microsoft.com/aspnet/core/security/authentication/certauth?view=aspnetcore-3.1) should that move?

Any further content on `dotnet dev-certs` should be in the [CLI area](https://docs.microsoft.com/dotnet/core/tools/).

Reply:
@BillWagner @blowdart @Rick-Anderson 

PR in draft for generating self-signed certs:

https://github.com/dotnet/docs/pull/21643

Reply:
Thanks @BillWagner for the merge.

FYI @Rick-Anderson looks like that one is ready, so what we could do is link to it in this doc.